x-pack/plugins/security_solution/public/management/cypress/tsconfig.json

```diff
@@ -22,6 +22,8 @@
       "path": "../../../tsconfig.json",
       "force": true
     },
-    "@kbn/security-plugin"
+    "@kbn/security-plugin",
```

Comment:
Auto-commit seems to have changed these

Reply:
Should i roll these changes back, not sure why this happened

Reply:
I had the same in my PR, should we ask the team why the CI auto commit changed this file?

Reply:
Here the comment from the team https://github.com/elastic/kibana/pull/148702#issuecomment-1378560904